src/HeadManager.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace P4\MasterTheme;
+
+/**
+ * Class HeadManager
+ *
+ */
+class HeadManager
```

Comment:
I don't think this class is called anywhere 🤔, so might be the reason the code isn't loaded